### PR TITLE
XQ100 -- Ingested

### PR DESCRIPTION
**Pull request by profxj**
As titled.

Includes a Notebook and docs

Will be added to v02 shortly.

### CODE REVIEW

File: igmspec/ingest/xq100.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    meta = Table()
+    for key in ['RA', 'DEC', 'zem', 'sig_zem', 'flag_zem']:
+        meta[key] = xq100_meta[key]
+    meta['STYPE'] = [str('QSO')]*nqso
+    # Return
+    return meta
+
+
+def hdf5_adddata(hdf, IDs, sname, debug=False, chk_meta_only=False):
+    """ Append GGG data to the h5 file
```
Comment (ntejos) [inline igmspec/ingest/xq100.py]: GGG -> XQ100

**Comment by ntejos** [comment]
The notebook is for HST_z2...

File: igmspec/ingest/xq100.py (hunk 1 above)
Comment (profxj) [inline igmspec/ingest/xq100.py]: Edited

File: igmspec/ingest/xq100.py [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+    # Add zem
+    xq100_meta['zem'] = xq100_meta['Z_QSO']
+    xq100_meta['sig_zem'] = xq100_meta['ERR_ZQSO']
+    xq100_meta['flag_zem'] = [str('XQ-100')]*nspec
+    # DATE-OBS
+    meanmjd = []
+    for row in xq100_meta:
+        gdm = row['MJD_OBS'] > 0.
+        meanmjd.append(np.mean(row['MJD_OBS'][gdm]))
+    t = Time(meanmjd, format='mjd', out_subfmt='date')  # Fixes to YYYY-MM-DD
```
Comment (profxj) [inline igmspec/ingest/xq100.py]: Confirmed.

**Comment by profxj** [comment]
There is an XQ-100 notebook.  I'll ingest that into the docs 
better in a different PR.

**Comment by profxj** [comment]
Merging.

**Pull request merged**